pkg/cli/clierror/cli_error.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2021 The Cockroach Authors.
```

Comment:
nit: should this be called clierror.go, to match the package name?

Reply:
There are more files coming to it in later PRs, including an `error.go` because there will be an `Error` type. That's the "native" one here.